lib/ConstExtract/ConstExtract.cpp

```diff
@@ -244,7 +259,12 @@ bool writeAsJSONToFile(const std::vector<ConstValueTypeInfo> &ConstValueInfos,
                              PropertyDecl->isStatic() ? "true" : "false");
               JSON.attribute("isComputed",
                              !PropertyDecl->hasStorage() ? "true" : "false");
-              JSON.attribute("value", toString(PropertyInfo.Value.get()));
+              auto value = PropertyInfo.Value.get();
```

Comment:
Maybe we should move this into the `if (valueKind != CompileTimeValue::ValueKind::Runtime)` code block to keep it closer to where it is used?

Reply:
I considered that, but since we also use `value` to get the kind, I opted to avoid computing the result twice.